### PR TITLE
Improve eeprom access reliability

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
Signed-off-by: Andriy Moroz <c_andriym@mellanox.com>

**- What I did**
 Updated eeprom plugin to retry when failed (5 retries)

**- How I did it**
Check if eeprom available. If not - wait 1 sec and retry

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/eeprom.py [hunk 1]
```diff
@@ -17,16 +17,37 @@
...
+
 class board(eeprom_tlvinfo.TlvInfoDecoder):
 
     _TLV_INFO_MAX_LEN = 256
+    RETRIES = 5
 
     def __init__(self, name, path, cpld_root, ro):
-        self.eeprom_path = "/bsp/eeprom/sys_eeprom"
+        for attempt in range(self.RETRIES):
+            if not os.path.islink(EEPROM_SYMLINK):
```
Comment (qiluo-msft) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/eeprom.py]:
```suggestion
            if not (os.path.exists(EEPROM_SYMLINK) or os.path.isfile(CACHE_FILE)):
```
like below if-statement?

**Comment by qiluo-msft** [comment]
How do you test in DUT?
As we discussed in email, could you help permanently disable the EEPROM driver, and start the box from clean and load_minigraph?

**Comment by andriymoroz-mlnx** [comment]
> How do you test in DUT?
> As we discussed in email, could you help permanently disable the EEPROM driver, and start the box from clean and load_minigraph?
> Another test case: follow the above one, reboot, and enable the driver, test the system recover.

Test#1: No mac - system keeps restarting services
Test#2 System recovered successfully after reboot + load_minigraph

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by qiluo-msft** [comment]
@lguohan `vsimage` test failed because of there is no `platform/vs/kvm-image.mk` on 201803 branch. Can we remove it from the PR check list for this branch?

**Pull request merged**